components/molecule/stepper/src/Step/settings.js

```diff
@@ -26,6 +26,13 @@ export const getIcon = ({
 
 export const getLabel = ({label, steps, step, design, current}) => {
   if (design === DESIGN.DEFAULT) return label
-  else if (design === DESIGN.BASIC) return label
-  else if (design === DESIGN.COMPRESSED && current) return `${step}/${steps}${label ? `: ${label}` : ''}`
+  if (design === DESIGN.BASIC) return label
+  if (design === DESIGN.COMPRESSED && current)
+    return (
+      <>
+        {`${step}/${steps}`}
+        {label && `: `}
```

Comment:
If `label` is `undefined` we will have an `"undefined"`, could it be?

Reply:
https://github.com/SUI-Components/sui-components/pull/2695#issuecomment-1930031816